libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -1835,8 +1835,8 @@ namespace webassembly {
           * Host function to calculate the pairing of (G1, G2) pairs on the elliptic curve bls12-381
           *
           * @ingroup crypto
-          * @param g1_points - a span containing a list of G1 points (P0, P1, P2... Pn) - 96*n bytes little-endian.
-          * @param g2_points - a span containing a list of G2 points (P0, P1, P2... Pn) - 192*n bytes little-endian..
+          * @param g1_points - a span containing a list of G1 points (P0, P1, P2... Pn) - affine coordinates 96*n bytes little-endian.
+          * @param g2_points - a span containing a list of G2 points (P0, P1, P2... Pn) - affine coordinates 192*n bytes little-endian..
           * @param n - the number of elements in the lists.
           * @param[out] result - the result e(g1_0, g2_0) * e(g1_1, g2_1) * ... * e(g1_n, g2_n) - 576 bytes little-endian.
```

Comment:
specify `affine coordinates` here.

Reply:
The result is just little-endian, not affine. At least it uses `toBytesLE` not `toAffineBytesLE`.

Reply:
Is this the right choice? Not even clear to me what the result is? Three g2 values?

Reply:
The result is `fp12`.

Reply:
Oh yes I see that now, thanks.